### DISCUSSION TITLE
2024年3月11日已经付费，但却显示需要升级，想要升级却没有获得25位激活码，付费邮箱 845052929@qq.com

### DISCUSSION THREAD

**Discussion by wuchunhua512** [getlantern/lantern-forum-cn#229] [Q&A]
2024年3月11日已经付费，但却显示需要升级，想要升级却没有获得25位激活码，付费邮箱 845052929@qq.com

**Comment by wuchunhua512** [comment]
@Mike777Khoo 附件是我们的支付账单。
![WechatIMG13558](https://github.com/getlantern/lantern-forum-cn/assets/17310126/947fd0a3-5be1-47fe-afaa-915c0da42380)

**Comment by MaxMKMax** [comment]
@wuchunhua512 您好。您能检查一下你的电子邮件和垃圾邮件吗？有一个不是 25 位数字的激活码。感觉25位激活码让用户感到困惑。它应该仅适用于销售我们产品的联属合作伙伴，并且购买的用户才需要这个激活码。对于您的情况，您不需要 25 位数的激活码。


或者您可以尝试重新登录。如果这不起作用，那么我将帮助您安排我们的团队来帮助您检查您的问题。谢谢!

**Comment by wuchunhua512** [comment]
@Mike777Khoo 你好，我的电子邮件和垃圾邮件，都没有收到邮件哦

![WechatIMG1430](https://github.com/getlantern/lantern-forum-cn/assets/17310126/0e86baa3-cdeb-48fb-afc9-541771a09d45)
![WechatIMG1431](https://github.com/getlantern/lantern-forum-cn/assets/17310126/d88bda73-2127-4b17-8a92-edf6787bb6fc)

**Comment by wuchunhua512** [comment]
@Mike777Khoo 我重新登录好几次，都是显示免费版，请安排技术团队来帮助检查我的问题。谢谢!

**Reply by MaxMKMax** [reply]
@wuchunhua512 
好的! 马上安排! 我会帮您通知团队紧急处理您的案件。请给我们一天的时间。感谢您的理解和等待。

**Reply by wuchunhua512** [reply]
@Mike777Khoo 好的，期待你们的回复，辛苦啦！

**Reply by MaxMKMax** [reply]
@wuchunhua512 亲, 您客气了. 不辛苦! 给你们最好的服务是我们的责任! 谢谢您对蓝灯的支持! 希望您可以在应用商店中给我们 5 星评级!

**Reply by wuchunhua512** [reply]
@Mike777Khoo 麻烦帮问问技术大佬们，我的问题搞定没？在线等着用！

**Reply by MaxMKMax** [reply]
@wuchunhua512 Hi. 您好. 明早之前就会给您搞定. 不好意思让您等了.

**Reply by wuchunhua512** [reply]
@Mike777Khoo 收到邮件啦，已正常激活专业版，可以正常使用了，非常感谢！辛苦啦！

**Reply by MaxMKMax** [reply]
@wuchunhua512 
下午好! 亲, 不客气呢! 非常感谢您对蓝灯的支持. 我们很开心又像您这么尊贵的客户.

**Reply by kisskof** [reply]
付费订单号：2023111827439
重新下载的客户端，账号登录不上。订单内的25位激活码没有。
我只能根据验证码来登录了，但是我需要登录的验证码一直都没有收到。请问怎么办？
另外是设备授权又无法使用，因为我只有这一台电脑设备登录过。请帮助一下我的邮箱是：client_2008@163.com 登录验证码给我一个可以吗？一直都没有收到包括垃圾邮件也没有。

**Comment by daisytan7980** [comment]
我也是，2.3付过费，没有邮件，直接用支付宝的，到现在没收到激活码，mytanxx@gmail.com 
![Uploading 0d7731f47848817c618b38e675efbe1.jpg…]()

**Reply by MaxMKMax** [reply]
@daisytan7980 您好! 在您的另一个帖子回复您了. 已经帮您处理着了. 不好意思等等一下哦.

**Reply by MaxMKMax** [reply]
@assholeyao 好的. 我帮您安排.

**Comment by hhmxianok** [comment]
你好，我也是开通了专业版但是还是显示免费版，请帮我处理一下，开通邮箱为1577257310@qq.com @Mike777Khoo

**Reply by hhmxianok** [reply]
![f72e76a1ee150f82cfc85958fc52be7](https://github.com/getlantern/lantern-forum-cn/assets/164504290/208db652-b519-4beb-8efe-028dc9d91af6)
这是邮箱截图

**Reply by MaxMKMax** [reply]
@wuchunhua512 你好。请问您是否尝试登录专业帐户？当您点击设备的“登录专业版”时，您将在邮件收到激活码。
https://lantern.io/zh-Hans/faq
您可以阅读此链接，它将帮助您解决登录问题。

同时, 想请问一下您用的是哪个系统? 是iOS还是Mac?

**Reply by wuchunhua512** [reply]
@Mike777Khoo 您好，怎么回事我电脑的蓝灯这几天突然就崩了，每次打开都直接闪退？我授权给其他两台设备的蓝灯也是打开就闪退！

**Reply by wuchunhua512** [reply]
@Mike777Khoo 我的主账号用的是Mac，授权给另外的账号是Windows10，麻烦帮看看是什么原因导致我电脑的蓝灯都是闪退。我的账号是专业版的了。

**Reply by wuchunhua512** [reply]
@Mike777Khoo 
Mac 下的蓝灯闪退问题，找到临时解决办法了：
删除 /Users/xxxx(自己mac的用户名)/Library/Application Support/Lantern/global.yaml
重新启动，然后重新授权即可

**Comment by daisytan7980** [comment]
为什么付过费以后， 流量用完的， 用了25位注册码，然后又变成这种了， 说是流量不够的。 
<img width="929" alt="微信图片_20240324152511" src="https://github.com/getlantern/lantern-forum-cn/assets/148767582/ea955a6e-e0df-4b83-984d-19eec8014e43">

**Reply by MaxMKMax** [reply]
@daisytan7980 您好! 您应该是没登陆专业版成功. 

或者另一个可能性就是您的帐号没被升级成功.

您试试看重新登录. 如果还不能您再回来找我, 我看能帮您什么忙.

**Comment by daisytan7980** [comment]
前几天还可以用的。是显示专业版。 直接点一下就可以， 今天一下子就不可以了， 显示流量用完了。

**Reply by MaxMKMax** [reply]
@daisytan7980 应该是被强制退出专业版帐号了. 需要您重新登录. 您试试看.

**Comment by kisskof** [comment]
重新下载的客户端，账号登录不上。订单内的25位激活码没有。
我只能根据验证码来登录了，但是我需要登录的验证码一直都没有收到。请问怎么办？
另外是设备授权又无法使用，因为我只有这一台电脑设备登录过。
请帮助一下我的邮箱是：client_2008@163.com登录验证码给我一个可以吗？一直都没有收到包括垃圾邮件也没有。

**Reply by kisskof** [reply]
付费订单号：2023111827439

**Comment by kennylo47** [comment]
我的macbook是升级蓝灯到7.8.8 变成了好像ios里面的界面，然后和上面的朋友一样没有激活码无法变成专业版
我在6月6日刚刚续费2年，之前都是自动变成专业版的 也已经写了邮件去询问 没回
注册邮件 lawy2k47@hotmail.com 谢谢

**Reply by MaxMKMax** [reply]
@kennylo47 已经帮您处理了. 请查看.

**Comment by kennylo47** [comment]
请问是如何为我处理？
没有看到激活码啊
是否发错邮箱？
***@***.***
在论坛我也发了续费付款截图 2024年6月6日付款的2年
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年6月19日 1:37
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: kennylo47 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 2024年3月11日已经付费，但却显示需要升级，想要升级却没有获得25位激活码，付费邮箱 ***@***.*** (Discussion #229)


@kennylo47<https://github.com/kennylo47> 已经帮您处理了. 请查看.

―

**Comment by liangzi16** [comment]
@Mike777Khoo,你好， 我也是2024年7月2日付款，一直未收到激活码，邮箱即本邮箱39053595@qq.com,付款截图见图，邮箱里垃圾箱、收件箱都已经查看过，一直未收到，麻烦帮处理，谢谢
![微信图片_20240704104327](https://github.com/getlantern/lantern-forum-cn/assets/174672932/ce8323fe-5381-4178-8e0b-c642f60f423f)
